modules/schedulers/src/test/java/org/mule/test/modules/schedulers/cron/StoppedCronSchedulerTestCase.java

```diff
@@ -60,8 +60,8 @@ private void runSchedulersOnce() throws Exception {
     flow.start();
     try {
       MessageSource flowSource = flow.getMessageSource();
-      if (flowSource instanceof SchedulerMessageSource) {
-        ((SchedulerMessageSource) flowSource).performPoll();
+      if (flowSource instanceof DefaultSchedulerMessageSource) {
+        ((DefaultSchedulerMessageSource) flowSource).run();
```

Comment:
Any reason to use `run()` and the the more direct `trigger()` here? (assuming i understood difference)